diagnostics/physics/ssh_eval.py

```diff
@@ -1,163 +1,72 @@
-    )
-    #satellite_ssh_index, satellite_ssh_points = compute_gs(satellite['adt'])
-    #satellite_ssh_points.to_netcdf('../data/obs/satellite_ssh_points.nc')
-    #satellite_ssh_index.to_pickle('../data/obs/satellite_ssh_index.pkl')
-    #read pre-calculate satellite_ssh_index and points
-    satellite_ssh_points = xarray.open_dataset('../data/obs/satellite_ssh_points.nc')
-    satellite_ssh_index = pd.read_pickle('../data/obs/satellite_ssh_index.pkl')
-
-    model_rolled = model_ssh_index.rolling(25, center=True, min_periods=25).mean().dropna()
-    satellite_rolled = satellite_ssh_index.rolling(25, center=True, min_periods=25).mean().dropna()
-
-    corr = pd.concat((model_ssh_index, satellite_ssh_index), axis=1).corr().iloc[0, 1]
-    corr_rolled = pd.concat((model_rolled, satellite_rolled), axis=1).corr().iloc[0, 1]
-
-    # Plot of Gulf Stream position and index based on SSH,
-    # plus position based on T200.
-    fig = plt.figure(figsize=(10, 6), tight_layout=True)
-    gs = gridspec.GridSpec(2, 2, hspace=.25)
-
-    ax = fig.add_subplot(gs[0, 0], projection=PC)
-    ax.add_feature(_LAND_50M)
-    ax.contour(model_grid.geolon, model_grid.geolat, model_t200, levels=[15], colors='r')
-    ax.contour(glorys_t200.longitude, glorys_t200.latitude, glorys_t200, levels=[15], colors='k')
-    add_ticks(ax, xlabelinterval=5)
-    ax.set_extent([-82, -50, 25, 41])
-    ax.set_title('(a) Gulf Stream position based on T200')
-    custom_lines = [Line2D([0], [0], color=c, lw=2) for c in ['r', 'k']]
-    ax.legend(custom_lines, ['Model', 'GLORYS12'], loc='lower right', frameon=False)
-
-    ax = fig.add_subplot(gs[0, 1], projection=PC)
-    ax.add_feature(_LAND_50M)
-    ax.plot(model_ssh_points.lon-360, model_ssh_points, c='r')
-    ax.plot(satellite_ssh_points.lon-360, satellite_ssh_points['__xarray_dataarray_variable__'], c='k')
-    add_ticks(ax, xlabelinterval=5)
-    ax.set_extent([-82, -50, 25, 41])
-    ax.set_title('(b) Gulf Stream position based on SSH variance')
-    ax.legend(custom_lines, ['Model', 'Altimetry'], loc='lower right', frameon=False)
-
-    ax = fig.add_subplot(gs[1, :])
-    model_ssh_index.plot(ax=ax, c='#ffbbbb', label='Model')
-    satellite_ssh_index.plot(ax=ax, c='#bbbbbb', label=f'Altimetry (r={corr:2.2f})')
-    model_rolled.plot(ax=ax, c='r', label='Model rolling mean')
-    satellite_rolled.plot(ax=ax, c='k', label=f'Altimetry rolling mean (r={corr_rolled:2.2f})')
-    ax.set_title('(c) Gulf Stream index based on SSH variance')
-    ax.set_xlabel('')
-    ax.set_ylim(-3, 3)
-    ax.set_ylabel('Index (positive north)')
-    ax.legend(ncol=4, loc='lower right', frameon=False, fontsize=8)
-
-    save_figure('gulfstream_eval', label=label, pdf=True)
+    # Set projection of each grid in the plot
+    # For now, sst_eval.py will only support a projection for the arctic and a projection for all other domains
+    if config['projection_grid'] == 'NorthPolarStereo':
```

Comment:
Not a big deal, but in other scripts (e.g., `sst_eval.py`), you used `p` for the grid projection:
https://github.com/NOAA-GFDL/CEFI-regional-MOM6/blob/801a17692c9497c6abfa0fbf3db5d63d8629350e/diagnostics/physics/sst_eval.py#L51-L54
It might be a good idea to make the naming consistent with similar scripts. I understand that if you do this, you'll also have to modify the following: https://github.com/NOAA-GFDL/CEFI-regional-MOM6/blob/6101793cfa12d34269fd8b1ee3d39fe3d233a20e/diagnostics/physics/ssh_eval.py#L95-L96 and https://github.com/NOAA-GFDL/CEFI-regional-MOM6/blob/6101793cfa12d34269fd8b1ee3d39fe3d233a20e/diagnostics/physics/ssh_eval.py#L102-L103 too.